INN/umbrella-sfpublicpress
# Subscribe widget

## Changes

This pull request makes the following changes:

- Adds the new San Francisco Public Press Subscribe widget and relevant styles

Desktop:
![Screen Shot 2020-02-04 at 2 50 36 PM](https://user-images.githubusercontent.com/18353636/73781234-e18f0a80-475d-11ea-9a66-d6ec2ec8a468.png)

Mobile:
![Screen Shot 2020-02-04 at 2 50 44 PM](https://user-images.githubusercontent.com/18353636/73781233-e18f0a80-475d-11ea-888c-d8905b7d67d2.png)

## Why



For #16 

## Testing/Questions

Features that this PR affects:

- Subscribe widget


Questions that need to be answered before merging:


Steps to test this PR:

1. Install the Pico wordpress plugin
2. Add in the Publisher ID and API Key for the account. Pico login info is in 1Pass under `SFPP Pico Login`
3. Add the new widget to the Homepage Top Right widget area and paste this as the embed code:
```
<form class="PicoSignupForm">
  <input type="email" placeholder="Email">
  <button type="submit"></button>
</form>
```
4. View it on desktop and mobile and make sure it all functions ok
